src/main/resources/public/raml/subscription.raml

```diff
@@ -14,39 +14,43 @@
       body:
         application/json:
           schema: subscriptionList
+      responses:
+        200:
+          body:
+            application/json:
+              schema: subscriptionResponse
 
     put:      
-      description: Modify an existing Subscription. 
+      description: Modify an existing Subscriptions.
       body:
         application/json:
           schema: subscriptionList
       responses:
         200:        
           body:
             application/json:
-              schema: subscriptionResponse            
+              schema: subscriptionResponse
                 
-    /{subscriptionName}:    
+    /{subscriptionNames}:
       uriParameters:
-        subscriptionName:
-          displayName: SubscriptionName
-          description: |
-            The name of the Subscription for info
-            required: true
-            type: string
+        subscriptionNames:
+          displayName: SubscriptionNames
+          description: The names of the Subscriptions for info
```

Comment:
Perhaps add in description about multiple subscription names being separated by comma?